src/benchmark/join_order_benchmark.cpp

```diff
@@ -55,26 +65,52 @@ int main(int argc, char* argv[]) {
       return 0;
     }
 
-    query_path = cli_parse_result["queries"].as<std::string>();
-    table_path = cli_parse_result["tables"].as<std::string>();
+    query_path = cli_parse_result["query_path"].as<std::string>();
+    table_path = cli_parse_result["table_path"].as<std::string>();
+    queries_str = cli_parse_result["queries"].as<std::string>();
 
     benchmark_config = std::make_shared<BenchmarkConfig>(CLIConfigParser::parse_basic_cli_options(cli_parse_result));
   }
 
+  // Check that the options "query_path" and "table_path" were specified
+  if (query_path.empty() || table_path.empty()) {
+    std::cerr << "Need to specify --query_path=path/to/queries and --table_path=path/to/tables" << std::endl;
+    std::cerr << cli_options.help({}) << std::endl;
+    return 1;
+  }
+
   /**
    * Use a Python script to download and unzip the IMDB. We do this in Python and not in C++ because downloading and
-   * unzipping is straight forward in Python.
+   * unzipping is straight forward in Python (and we suspect in C++ it might be... cumbersome).
    */
   const auto setup_imdb_command = "python3 scripts/setup_imdb.py "s + table_path;
   const auto setup_imdb_return_code = system(setup_imdb_command.c_str());
-  Assert(setup_imdb_return_code == 0, "setup_imdb.py failed");
+  Assert(setup_imdb_return_code == 0, "setup_imdb.py failed. Did you run the benchmark from the project root dir?");
 
-  // The join-order-benchmark ships with these, but we do not want to run them (and hyrise can't, as a matter of fact)
+  // The join-order-benchmark ships with these two .sql scripts, but we do not want to run them as part of the benchmark
+  // as they do not contains actual queries
   const auto non_query_file_names = std::unordered_set<std::string>{"fkindexes.sql", "schema.sql"};
 
   benchmark_config->out << "- Benchmarking queries from " << query_path << std::endl;
   benchmark_config->out << "- Running on tables from " << table_path << std::endl;
 
+  std::optional<std::unordered_set<std::string>> query_subset;
+  if (queries_str == "all") {
+    benchmark_config->out << "- Running all queries from specified path" << std::endl;
+  } else {
+    benchmark_config->out << "- Running subset of queries: " << queries_str << std::endl;
+
+    // "a, b, c, d" -> ["a", " b", " c", " d"]
```

Comment:
This is something that all current benchmarks should share. Could you move that into a helper method?